src/content/en/2020/privacy.md

```diff
@@ -12,12 +12,166 @@ translators: []
+  description="This figure shows the percentage of websites that load at least one tracker.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1325818112&format=interactive",
+  sheets_gid="1591448294"
+  )
+}}
+
+We examine the largest (most widely used) trackers and plot the prevalence of the 10 most popular ones.
+
+{{ figure_markup(
+  image="biggest_third-party_trackers.png",
+  caption="Top 10 Trackers",
+  description="This figure shows the prevalence of the 10 most popular trackers used on mobile and desktop clients.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=850649042&format=interactive",
+  sheets_gid="1677398038"
+  )
+}}
+
+The largest player on the online tracking market is definitely Google, with eight of  its tracking domains present in the top 10 trackers and prevalent on at least 70% of websites. Following are Facebook and Cloudflare. 
+
+WhoTracksMe’s tracker list also defines categories that the trackers belong to. The following figure shows the distribution of the different categories for the 100 largest trackers.
+
+{{ figure_markup(
+  image="tracker_categories.png",
+  caption="Categories of the 100 most popular trackers",
+  description="This figure shows distribution of the top 100 trackers on the web.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1117413918&format=interactive",
+  sheets_gid="1431872451",
+  )
+}}
+
+Nearly 60% of the most popular trackers are advertising-related. This could be due to the profitability of the online advertising market.
+
+### Cookies
+We looked into the most popular cookies being set on websites in HTTP's response header, according to their name and domain.
+
+<figure markdown>
+| Client | Domain | Cookie Name | Websites present on   |
+| -------- | ------- | ------ | ------ |
+| desktop |	doubleclick.net|	test_cookie|	24%|
+|desktop|	facebook.com|	fr|		10%|
+|desktop|	youtube.com|	VISITOR_INFO1_LIVE|	10%
+|desktop|	youtube.com|	YSC|10%
+|desktop|	doubleclick.net|	IDE|9%
+|desktop|	doubleclick.net|unknown|	9%
+|desktop|	youtube.com	|GPS|9%
+|desktop|	doubleclick.net|	unknown|	8%
+|desktop|	google.com|	NID|	6%
+|desktop|	doubleclick.net|	unknown|6%
+|mobile|	doubleclick.net|	test_cookie|32%
+|mobile|	doubleclick.net	|IDE|	21%
+|mobile|	facebook.com|	fr|	10%
+|mobile|	youtube.com|	VISITOR_INFO1_LIVE|10%
+|mobile	|youtube.com|	YSC|10%
+|mobile|	google.com|	NID|10%
+|mobile|	youtube.com	|GPS|	8%
+|mobile|	doubleclick.net	|DSID|	7%
+|mobile|	yandex.ru|	yandexuid|	6%
+|mobile|	yandex.ru|	i|	6%
+
+<figcaption>{{ figure_link(caption="Top cookies", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
+</figure>
```

Comment:
Suggest splitting this into two tables, and also using HTML so can apply numeric class and align the percentages.

```suggestion
<figure>
  <table>
    <thead>
      <tr>
        <th>Domain</th>
        <th>Cookie Name</th>
        <th>Websites present on</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>doubleclick.net</td>
        <td>test_cookie</td>
        <td class="numeric">24%</td>
      </tr>
      <tr>
        <td>facebook.com</td>
        <td>fr</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>VISITOR_INFO1_LIVE</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>YSC</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>IDE</td>
        <td class="numeric">9%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>unknown</td>
        <td class="numeric">9%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>GPS</td>
        <td class="numeric">9%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>unknown</td>
        <td class="numeric">8%</td>
      </tr>
      <tr>
        <td>google.com</td>
        <td>NID</td>
        <td class="numeric">6%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>unknown</td>
        <td class="numeric">6%</td>
      </tr>
    </tbody>
  </table>
  <figcaption>{{ figure_link(caption="Top cookies on desktop sites", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
</figure>

<figure>
  <table>
    <thead>
      <tr>
        <th>Domain</th>
        <th>Cookie Name</th>
        <th>Websites present on</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>doubleclick.net</td>
        <td>test_cookie</td>
        <td class="numeric">32%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>IDE</td>
        <td class="numeric">21%</td>
      </tr>
      <tr>
        <td>facebook.com</td>
        <td>fr</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>VISITOR_INFO1_LIVE</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>YSC</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>google.com</td>
        <td>NID</td>
        <td class="numeric">10%</td>
      </tr>
      <tr>
        <td>youtube.com</td>
        <td>GPS</td>
        <td class="numeric">8%</td>
      </tr>
      <tr>
        <td>doubleclick.net</td>
        <td>DSID</td>
        <td class="numeric">7%</td>
      </tr>
      <tr>
        <td>yandex.ru</td>
        <td>yandexuid</td>
        <td class="numeric">6%</td>
      </tr>
      <tr>
        <td>yandex.ru</td>
        <td>i</td>
        <td class="numeric">6%</td>
      </tr>
    </tbody>
  </table>
  <figcaption>{{ figure_link(caption="Top cookies on mobile sites", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
</figure>
```